Added a speedometer

MelgMKW opened the pull request:
Added:
-A speedometer that works in all modes
-Documentation for screens and screenelements. Improved some of the headers.

SeekyCt commented:
Closing since it was manually merged in d24a9e073063f48a6715c9bc2a912ed00e8554fa

MelgMKW committed:
Added a speedometer

Added:
-A speedometer that works in all modes
-Documentation for screens and screenelements. Improved some of the headers.
